src/utils/navigateToWebsite.js

```diff
@@ -0,0 +1,5 @@
+import config from '../../config';
+
+const navigateToWebsite = () => { window.location.href = `${config.env.masterbornWebsite}/career` };
```

Comment:
Shouldn't it be called navigateToCarrier or navigateToWebsiteCarrier?

Reply:
also, let's use navigate from gatsby.
https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/#how-to-use-the-navigate-helper-function